spring-pulsar-spring-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarAnnotationDrivenConfiguration.java

```diff
@@ -62,8 +65,9 @@ ConcurrentPulsarListenerContainerFactory<?> pulsarListenerContainerFactory(
 		PulsarProperties.Listener listenerProperties = this.pulsarProperties.getListener();
 		map.from(listenerProperties::getSchemaType).to(containerProperties::setSchemaType);
 		map.from(listenerProperties::getAckMode).to(containerProperties::setAckMode);
-		map.from(listenerProperties::getBatchTimeoutMillis).to(containerProperties::setBatchTimeoutMillis);
-		map.from(listenerProperties::getMaxNumBytes).to(containerProperties::setMaxNumBytes);
+		map.from(listenerProperties::getBatchTimeout).asInt(Duration::toMillis)
```

Comment:
@cbornet good pragmatic call on stopping the conversion here (ie leave the downstream things still using the previous types) as that would have introduced quite the ripple. We can adjust those usages as we re-encounter those code areas (eg the ListenerContainerProperties in this case).